src/components/LayoutTree.tsx

```diff
@@ -881,6 +881,41 @@ const LayoutTree = () => {
   return (
     <div
       className={css({
+        // the hideCaret animation must run every time the indent changes on iOS Safari, which necessitates replacing the animation with an identical substitute with a different name
+        animation:
```

Comment:
@raineorshine This works, but is pretty unpleasant to look at. Any attempt that I made to package this code or make it dynamic (including an inlined IIFE) broke Panda's static analysis. I'll try to poke at it some more later to see if I can get it to work better. If you have a preferred way to abstract logic like this while keeping static analysis, please share.

Reply:
Yes, good point. I'm thinking either a recipe with variants would work (possibly programmatically generated), or switching to a `style` attribute and accessing the animation through `token`.

Reply:
@yangchristina Could I get your input on this? Would recipe variants be able to select on the indentation level at runtime? 

We have to trick the browser into thinking the animation has changed at each indentation level, so that's why each animation has its own name.

Reply:
@raineorshine Yes, recipes should be fine to use, just make sure you add `staticCss: ['*']`. You can also use `cva` since this is only used in one place, and you won't need `staticCss: ['*']` for `cva` (it'll automatically generate all variants)

Reply:
@yangchristina Great, thanks for confirming!

@ethan-james Would you mind looking into [cva](https://panda-css.com/docs/concepts/recipes#atomic-recipe-or-cva) as a way to clean this up? Reach out to me personally if this looks like a lot of trouble, as I want to be respectful of your time.

Reply:
@yangchristina @raineorshine Thanks for your guidance! The latest commit represents how I think I want it to work, but it still doesn't seem to be generating the classes for the recipe variant. If you think there's a specific piece that I can tweak to make it work, please let me know.

Reply:
Now it's no longer the most recent commit. Better to just look at the whole diff.

Reply:
`hideCaretAnimationNames` must be in the same file as `cva`. `hideCaret` might also have to be directly in `LayoutTree.tsx` instead of imported.
I tried with both `hideCaret` and `hideCaretAnimationNames` in `LayoutTree.tsx` and it did generate the classes.

Reply:
You can check whether the classes have been generated with `npx panda debug src/components/LayoutTree.tsx`

Reply:
@yangchristina Thanks! Keeping them in a separate file seems to work, with the only issue being that Panda's config can't pull the constants from `getHideCaretAnimationName.ts`. It says `ERROR: Could not resolve "../../styled-system/css"` so I guess there are different stages of the toolchain or something like that. Duplicating the constants in `getCaret.config.ts` & `getHideCaretAnimationName.ts` seems like a relatively small price to pay.